### PR TITLE
Added changelog for 7.0.0 Beta1

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 7.0.0-beta1 - 2019-12-31
```
Comment (sagikazarmark) [inline CHANGELOG.md]: I'd love to start using https://keepachangelog.com/en/1.0.0/ format

Comment (Nyholm) [reply CHANGELOG.md]: Same, I've added some headings.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+
+The diff might look very big but 95% of Guzzle users will be able to upgrade without modification. 
+Please see [the upgrade document][UPGRADING.md] that describes all BC breaking changes.   
+
+### Added
+
+* Implement PSR-18 [#2421](https://github.com/guzzle/guzzle/pull/2421) [#2474](https://github.com/guzzle/guzzle/pull/2474)
+* Adding PHP 7 types [#2442](https://github.com/guzzle/guzzle/pull/2442) [#2449](https://github.com/guzzle/guzzle/pull/2449) [#2466](https://github.com/guzzle/guzzle/pull/2466) [#2497](https://github.com/guzzle/guzzle/pull/2497) [#2499](https://github.com/guzzle/guzzle/pull/2499)
```
Comment (sagikazarmark) [inline CHANGELOG.md]:
```suggestion
* PHP 7 types [#2442](https://github.com/guzzle/guzzle/pull/2442) [#2449](https://github.com/guzzle/guzzle/pull/2449) [#2466](https://github.com/guzzle/guzzle/pull/2466) [#2497](https://github.com/guzzle/guzzle/pull/2497) [#2499](https://github.com/guzzle/guzzle/pull/2499)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,32 @@
...
+* Add IDN support for redirects [2424](https://github.com/guzzle/guzzle/pull/2424)
```
Comment (sagikazarmark) [inline CHANGELOG.md]:
```suggestion
* IDN support for redirects [2424](https://github.com/guzzle/guzzle/pull/2424)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,32 @@
...
+* Dont allow passing null as third argument to `BadResponseException::__construct()` [#2427](https://github.com/guzzle/guzzle/pull/2427)
+* Use SAPI constant instead of method call [#2450](https://github.com/guzzle/guzzle/pull/2450)
+* Use native function invocation [#2444](https://github.com/guzzle/guzzle/pull/2444)
+* Better defaults for PHP installations with old ICU lib [2454](https://github.com/guzzle/guzzle/pull/2454)
+* Added visibility to all constants [#2462](https://github.com/guzzle/guzzle/pull/2462)
+* Dont allow passing `null` as URI to `Client::request()` and `Client::requestAsync()` [#2461](https://github.com/guzzle/guzzle/pull/2461)
+
+## Removed
+
+* Removed `RequestException::getResponseBodySummary()` [#2425](https://github.com/guzzle/guzzle/pull/2425)
```
Comment (sagikazarmark) [inline CHANGELOG.md]:
```suggestion
* `RequestException::getResponseBodySummary()` [#2425](https://github.com/guzzle/guzzle/pull/2425)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,32 @@
...
+* Removed `CookieJar::getCookieValue()` [#2433](https://github.com/guzzle/guzzle/pull/2433)
```
Comment (sagikazarmark) [inline CHANGELOG.md]:
```suggestion
* `CookieJar::getCookieValue()` [#2433](https://github.com/guzzle/guzzle/pull/2433)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,32 @@
...
+* Removed `uri_template()` and `UriTemplate` [#2440](https://github.com/guzzle/guzzle/pull/2440)
```
Comment (sagikazarmark) [inline CHANGELOG.md]:
```suggestion
* `uri_template()` and `UriTemplate` [#2440](https://github.com/guzzle/guzzle/pull/2440)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,32 @@
...
+* Remove deprecated request options `save_to` and `exceptions` [#2464](https://github.com/guzzle/guzzle/pull/2464)
```
Comment (sagikazarmark) [inline CHANGELOG.md]:
```suggestion
* Deprecated request options `save_to` and `exceptions` [#2464](https://github.com/guzzle/guzzle/pull/2464)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,36 @@
...
+### Fixed 
+
+* Logging when Promise rejected with a string [#2311](https://github.com/guzzle/guzzle/pull/2311)
+
+### Removed
+
+* `RequestException::getResponseBodySummary()` [#2425](https://github.com/guzzle/guzzle/pull/2425)
+* `CookieJar::getCookieValue()` [#2433](https://github.com/guzzle/guzzle/pull/2433)
+* `uri_template()` and `UriTemplate` [#2440](https://github.com/guzzle/guzzle/pull/2440)
+* Deprecated request options `save_to` and `exceptions` [#2464](https://github.com/guzzle/guzzle/pull/2464)
```
Comment (gmponos) [inline CHANGELOG.md]: Removed Seek Exception.

Comment (gmponos) [inline CHANGELOG.md]: Would prefer it this way as it does not confuses if it is deprecated or removed.

```suggestion
* request options `save_to` and `exceptions` [#2464](https://github.com/guzzle/guzzle/pull/2464)
```

**Comment by Nyholm** [comment]
Thank you @gmponos for your review. I've updated accordingly.

**Comment by Nyholm** [comment]
Thank you for merging

**Pull request merged**